bin/runner.js

```diff
@@ -188,32 +188,35 @@ function writeFile(testsDocPath) {
 
     var testsDocModulePath = path.join(testsDocPath, modulePath.join("/"));
 
-    mkdirp(testsDocModulePath, function(err) {
-      if (err) {
+    mkdirp(testsDocModulePath)
+      .then(function () {
+        fs.writeFile(
```

Comment:
Might be easier to eventually rewrite this in async/await style, but that's perhaps for another PR...

Reply:
yes!